src/identity/domain/Person.js

```diff
@@ -320,6 +321,17 @@ const Person = attributes(
       return this.set('taxId', taxId)
     }
 
+    /**
+     * hides first digests of the taxId field
+     * @memberof module:domain.Person
+     * @returns {void}
+     */
+    hideTaxId () {
+      if (this.taxId) {
+        return this.set('taxId', mask(this.taxId))
```

Comment:
what is the return value of `mask()` function?

Reply:
mask('01-1234567');
// => XX-XXX4567

mask('011234567');
// => XXXXX4567